src/SdfEntityCreator.cc

```diff
@@ -280,6 +304,9 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Model *_model,
 
   // Links
   bool canonicalLinkCreated = false;
+  const auto[canonicalLink, canonicalLinkName] =
```

Comment:
nit: `canonicalLinkName` is not used, so can just be `const auto *canonicalLink = _model->CanonicalLink();`